Sourcery/main.swift

```diff
@@ -114,8 +117,9 @@ func runCLI() {
         Option<Path>("output", ".", description: "Path to output. File or Directory. Default is current path."),
         Option<Path>("config", ".", description: "Path to config file. Directory. Default is current path."),
         Argument<CustomArguments>("args", description: "Custom values to pass to templates.")
-    ) { watcherEnabled, disableCache, verboseLogging, prune, sources, templates, output, configPath, args in
+    ) { watcherEnabled, disableCache, verboseLogging, quite, prune, sources, templates, output, configPath, args in
         do {
+            Log.level = verboseLogging ? .verbose : quite ? .errors : .warnings
```

Comment:
I wonder if this works correctly without the `( )`, I'd recommend using something like switch for readability imho